public/build/entrypoints.json

```diff
@@ -2,59 +2,28 @@
   "entrypoints": {
     "app": {
       "js": [
-        "/build/runtime.c3e9a731.js",
-        "/build/755.5a8586e9.js",
-        "/build/31.ccf7320d.js",
-        "/build/169.60ab0201.js",
-        "/build/app.71537808.js"
-      ],
-      "css": [
-        "/build/169.14a1c34e.css",
-        "/build/app.ac992678.css"
-      ]
-    },
-    "login": {
-      "js": [
-        "/build/runtime.c3e9a731.js",
-        "/build/755.5a8586e9.js",
-        "/build/login.6b6b5d56.js"
+        "/build/runtime.js",
+        "/build/vendors-node_modules_core-js_internals_add-to-unscopables_js-node_modules_core-js_internals_a-029029.js",
```

Comment:
The generated file names have to be so long @weaverryan?

Reply:
That's a Webpack thing. If you did a "prod" build, the filenames are things like `0.js`, `1.js`. But in dev mode, Webpack creates long filenames to help give you an idea of what's in there.

About the size: I'm not sure - probably it just has a lot in it. It also, iirc, contains the sourcemap IN the file at the bottom. And, it's unminified because it's a dev build. But we could open up that file to make sure it doesn't contain any huge files that we didn't intend to import.